runtime/src/inflation_rewards/mod.rs

```diff
@@ -49,16 +50,17 @@ pub fn redeem_rewards(
             ));
         }
 
+        let mut stake = *stake;
```

Comment:
nice that we can copy a bit less memory with this change and get rid of the unwrap at the `redeem_rewards` call site